thereseLYR/3D-Model-Ecommerce-Frontend
# chore: updated model images

## Changes
- replaced image PNGs with real photos
- added additional model image for ghost model
- cleaned up manifest file to prevent errors on load
- changed objectFit of ModelBox to 'cover' to prevent image stretching

## How to test
- Navigate to the landing page and ensure that the model thumbnails reflect correctly



## Known Issues
-  N/A

## Screenshots (if any)
![image](https://user-images.githubusercontent.com/47405262/197583009-ad1ef246-1eb7-4bb7-9c59-4fac28556f2b.png)
